sdk/test/UnitTests/Custom/Runtime/DefaultConfigurationAutoModeResolverTests.cs

```diff
@@ -16,14 +16,14 @@
 using Amazon.Runtime;
 using Amazon;
 using Moq;
-using Xunit;
```

Comment:
This class was using xUnit, but the project is configured for MSTest so I think the tests weren't running... I updated it to match the other custom tests.